test/unit/testPaymentStatus.js

```diff
@@ -47,9 +53,8 @@ describe('PaymentStatus', function () {
         });
 
         it('Should not create an error if sendApplication succeeds', function (done) {
-
+            s2sAuthoriseStub.returns(Promise.resolve({name: 'Success'}));
```

Comment:
This stub should be mocking `services.submitApplication()` rather than `services.authorise()` because the `result` that `services.submitApplication()` returns will ensure that an error is not returned by `sendApplication()` and `formdata` is persisted.

You'll need to mock `services.saveFormData()` so it returns a valid success value and then `sendApplication()` should return the correct `errors` value (none in this case). In this case the app will log an error with `logger.info('Successfully persisted user data');` so you could `spy` on `logger` to ensure that `info()` is called on it with the correct value.

You could also write another test for the case that `services.saveFormData()` and therefore `persistFormData()` returns an error. In this case you'd need to do similar to the test above.